js/views-manager.js

```diff
@@ -12,33 +8,44 @@ var ViewsManager = {
   },
   _currentViewMeta: {
     viewName: "",
-    projectId: ""
+    projectId: "",
+    issue: ""
   },
   _projects: [],
   $controlsContainer: $("#project-container .controls"),
   $projectsContainer: $("#project-container .projects"),
+  $issueBtns: $(".issue-btn"),
   MessageView: {
     _$container: $("#message-view"),
-    setMessages: function(header,bodyText) {
-      this._$container.find("h2").html(header);
-      this._$container.find("p").html(bodyText);
-    },
-    show: function() {
+    show: function(header,bodyText) {
```

Comment:
I merged `ViewsManager.MessageView.setMessages` and `ViewsManager.MessageView.show` into one method: `ViewsManager.MessageView.show(header, bodyText)`